inference/embed-to-db-similar.py

```diff
@@ -0,0 +1,192 @@
+# Imports
```

Comment:
This new file looks very similar to text-to-db-similar.py so I'll throw a comment on each of the changes for ease of review